crates/orderbook/src/database/orders.rs

```diff
@@ -429,6 +430,13 @@ fn full_order_into_model_order(order: FullOrder) -> Result<Order> {
         sender: onchain_user,
         placement_error: onchain_placement_error,
     });
+    let executed_fee_amount = match &class {
+        OrderClass::Limit(LimitOrderClass {
+            executed_surplus_fee,
+        }) => *executed_surplus_fee,
+        OrderClass::Market | OrderClass::Liquidity => big_decimal_to_u256(&order.sum_fee)
```

Comment:
order.sum_fee is 0 for limit orders

Reply:
This changes the meaning of the field. Are we fine with that?
Usually unless otherwise specified `fee` always relates to a fee that was signed by the user with their order, right?

Reply:
That was my intention. To have a single `executed_fee` field that will represent the fee that was paid for gas costs.
I don't see a reason to have two different fields for this.

Reply:
I had a quick check and it doesn't look like the existing backend code base would break if we change that.
Also it doesn't appear to be stored in a DB anywhere (which would have made things weirder) so it really only seems to affect the frontend.
And for the frontend it's arguably more honest to show that we took some fees for executing the limit order.

Reply:
Frontend is currently showing the fee for limit orders, it's just reading it from the field I want to drop (the `executed_surplus_fee`).

Reply:
I asked them how they read that value and will wait for their confirmation before merging.
Although painful change, I still think this is a perfect timing to clean up the fees in our system before adding more (protocol fees etc).

Reply:
This looks like a 1:1 copy of the full_order_into_model_order method. Can we call the shared one from the orderbook crate to remove the duplication?

They do seem similar, one different thing is how the order status is calculated. Since I am not 100% sure we should merge these two, will do it in a separate PR so it could be reverted if needed.